app/src/main/java/com/serwylo/lexica/view/LexicaLogo.java

```diff
@@ -60,9 +61,11 @@ private void setupSoftwareCanvas() {
 		}
 	}
 
+	private final Rect textBounds = new Rect();
```

Comment:
As in `BoardView`, can this be inlined too?

Reply:
This one I don't mind inlining since the canvas is cached.